### ISSUE TITLE
Better Time Input?

### ISSUE THREAD

**Issue by rye** [kotct/docket-old#35]
Our current system simply outputs a default webkit date/time selection interface. While this fits the purpose, it does not fit with the design of the website.

I would recommend that we write a quick little JavaScript date constructor to operate on a group of fields, and then make a bunch of select fields as our replacement. I'd like to avoid using any jQuery modules, so I suppose I'll likely be the one to do this.

**Comment by cg505** [comment]
Why is this closed?

**Comment by rye** [comment]
Lack of activity.

**Comment by Sammidysam** [comment]
That is no reason to close an issue, unless you have completely given up on ever doing the work for this.

**Comment by rye** [comment]
There's plenty of reason. If there's no response, that indicates no interest. If there's no interest, that indicates that the idea is bad. If the idea is bad, then the issue can be closed.

**Comment by Sammidysam** [comment]
Not necessarily; it could be that the other contributors just did not get the time to look into it thoroughly.  In any case, the other contributors should say that it is a bad idea if they believe so.

**Comment by rye** [comment]
It's helpful to know, beyond a shadow of a doubt, why the other contributors aren't responding at all. Lack of response _is_ a sign of lack of interest. Heck, I don't know why we even use issues for recording ideas, since if no-one will pay attention for 5+ months that's plenty of time to just go ahead and implement it. It'd've been nice if this lack of ability to pay attention and respond was brought to my attention so that I could factor that into my response measurement. If it takes 5 months and a closure of the issue to actually initiate a response, then something's very wrong with the system.

**Comment by rye** [comment]
If we're trying to work synchronously (i.e. requesting others' approval before starting on implementation), then we need to actually communicate why we're not paying attention as opposed to shirking that. If we're trying to work asynchronously (i.e. thinking about something on our own and going and implementing it), which I would **_much**_ prefer, then we need to stop using issues and to-do lists and just get stuff done.

**Comment by Sammidysam** [comment]
I made some comments on Gitter that I should reflect here:

> I am kind of worried about the status of this program. I think we all have different ideas on where it is going, and it makes me uncomfortable. I don't think we set a clear plan of what the end result should look like when we began the program, and it has led to a sort of mess of ideas.
> Maybe my thought is just feeling bad for the lack of discussion that has occurred over the last few months. But I think our plan is not set in stone, and has led to conflicting ideas over time that need to be rectified.

I think that we need to all agree on a very set plan of what we want the end result to be, and then working asynchronously should work fine.

**Comment by rye** [comment]
Great. Since this issue is not the appropriate place for this, I will create a new issue for all discussion regarding this.

**Comment by cg505** [comment]
The reason for the lack of activity can be expressed in one word: school.

**Comment by Sammidysam** [comment]
For me, probably some other things, too.
